src/routes/(console)/organization-[organization]/settings/deleteOrganizationModal.svelte

```diff
@@ -94,12 +95,24 @@
                   }))
               };
 
-    $: upcomingInvoice = invoices?.invoices.find((i) => i.status === 'upcoming' && i.amount > 0);
-
     $: if (!showDelete) {
         // reset on close.
         organizationName = '';
     }
+
+    let getEstimate = async () => {
+        if(isCloud) {
+            try {
+                error = '';
+                estimation = await sdk.forConsole.billing.estimationDeleteOrganization(
+                    $organization.$id
+                );
+            } catch (e) {
+                error = e.message;
+                console.log(e);
+            }
+        }
+    };
```

Comment:
Let's use `function` instead of an arrow function

Reply:
updated